### ISSUE TITLE
Filtering (using Lanczos3 for example)

### ISSUE THREAD

**Issue by Rodrigodd** [jcupitt/vipsdisp#40]
I am currently working on integrating libvips into an image viewer that uses tiling. I am trying to use libvips specifically because it can filter the image during resizing (Lanczos, etc.) (See: https://github.com/Rodrigodd/image-decoder/pull/1).

I recently discovered this project, and I imagine it could serve as a good reference implementation, but it seems to be missing the filtering functionality when zooming out.

Could filtering be added to this project? If it's straightforward to implement, I can try adding it myself. I’ve actually already attempted this, but the code is a bit hard to follow due to the handling of pyramidal image formats.

As a workaround, I will check if I can generate pyramidal images with pre-applied filtering.

Edit: I just found that #1 discusses this. I will check out @aferrero2707's fork and see how it goes.

**Comment by jcupitt** [comment]
We have an enhancement issue for this now, so I'll close.

https://github.com/jcupitt/vipsdisp/issues/43
